CONTRIBUTING.md

```diff
@@ -8,7 +8,27 @@ Welcome to the TastyTrails project! We're excited to have you contribute and hel
 - Fork the repository and clone it to your local machine.
 - Create a new branch for your feature or bug fix: `git checkout -b feature-name` or `git checkout -b bugfix-name`.
 - Make sure to keep your fork updated by pulling from the `main` branch regularly.
+- You can follow the below video to setup your project locally (Instructions may vary for windows as i have set this up on Linux)
+- [How to setup project locally](https://youtu.be/eJ0p6rYek-8) | [How to generate github API key](https://www.youtube.com/watch?v=iLrywUfs7yU)
+- Here is a list of required packages to install as mentioned in the video
+  - **Mongo and its packages**: 
+  MongoDB Community Edition – https://www.mongodb.com/try/download/community
+  Mongodb shell – https://www.mongodb.com/try/download/shell
+  Mongodb compass – https://www.mongodb.com/try/download/compass
+  Make sure to check if mongo is up and running (You can refer youtube for troubleshooting)
+  - **Npm packages**: 
+    To setup initial required packages for both front-end and backend, go to respective folders i.e */frontend and /backend* and execute the below command
+    > `npm init`
```

Comment:
`npm init` is used to initiate npm.
I think this should be ` npm i install` to install the packages. Please Edit.